apps/glusterfs/volume_entry.go

```diff
@@ -36,6 +38,8 @@ const (
 	DEFAULT_EC_DATA               = 4
 	DEFAULT_EC_REDUNDANCY         = 2
 	DEFAULT_THINP_SNAPSHOT_FACTOR = 1.5
+
+	HEKETI_ARBITER_KEY = "x.heketi.arbiter"
```

Comment:
Gluster ignores(does not interpret any but stores them in gluster database) option that is form "user.*". I was thinking of using user.heketi-* as a namespace for all things heketi.

Something like user.heketi-arbiter=yes  OR user.heketi-arbiter=1, based on whether we want it to be a int or bool.

Reply:
Excellent, I was looking for something like that but couldn't find it in the gluster docs. I did try passing it the current key which was rejected. Does it have to have only one dot? If not I slightly prefer "user.heketi.arbiter" and I would go with any value but true-ness is indicated by any value that parses to true with go's [strconv.Parsebool](https://golang.org/pkg/strconv/#ParseBool).

Reply:
Verified in gluster code that user.heketi.* should work. I prefer dot as a separator too, just hadn't see options with dot with them in gluster.